indexing-service/src/main/java/io/druid/indexing/common/task/AbstractTask.java

```diff
@@ -176,6 +202,11 @@ public TaskStatus success()
     return TaskStatus.success(getId());
   }
 
+  public TaskStatus failure()
```

Comment:
not really needed..just added for sake of completeness as there is `success()` method